main.c

```diff
@@ -126,55 +124,38 @@ void server_loop(void *parameters) {
   csp_packet_t *packet;
 
   /* Create socket and listen for incoming connections */
-  printf("Under Test 1");
   sock = csp_socket(CSP_SO_NONE);
   csp_bind(sock, CSP_ANY);
   csp_listen(sock, 5);
   portBASE_TYPE err,err2;
 
   /* Super loop */
+  printf("Starting CSP server\n");
   for (;;) {
-    printf("Under Test Super Loop\n");
     /* Process incoming packet */
     if ((conn = csp_accept(sock, 10000)) == NULL) {
       /* timeout */
       continue;
     }
-    printf("Under Test Accept\n");
     while ((packet = csp_read(conn, 50)) != NULL) {
-      printf("Reading Packet\n");
       switch (csp_conn_dport(conn)) {
         case TC_HOUSEKEEPING_SERVICE:
-          printf("Read Packet: %s - %d\n", packet->data, packet->id);
           err = xQueueSendToBack(service_queues.hk_app_queue, packet,
                                  NORMAL_TICKS_TO_WAIT);
-          printf("333\n");
-          err2 = xQueueSendToBack(response_queue, conn,
```

Comment:
If remove this queue here, the ground station task may not run due to lack of conn. Do I still need to remove it?

Reply:
Oh, I see what you're doing now.
I think we'll need to change the response server a bit, since right now the connection is closed at the end of this main loop, hence it will be invalid very soon after this call is made.
I think we'll want to do 2 things:
1. send a response synchonously - that is as soon as the packet is recieved, send an acknowledgement on the same connection like you're doing here, but synchronously so that the connection still exists (we can do this later on)
2. Think a little bit more about how data (telemetry) responses should be sent.

for now I just removed it, we can revisit, I have another PR where I'll show you what I'm proposing.